partiql-conformance-test-generator/src/schema.rs

```diff
@@ -27,9 +27,10 @@ pub struct TestCase {
 /// Test case kind
 ///
 /// Currently, just supports `Parse` test cases. In the future, other test case variants will be
-/// added (e.g. evaluation, type-checking)
+/// added (e.g. evaluation, type-checking). For now, the other test case variants will be `Ignore`.
 pub enum TestCaseKind {
     Parse(ParseTestCase),
+    Ignore,
```

Comment:
As this is temporary, it might be good to name it something other than `Ignore`. Perhaps `WIP` or `NotYetImplemented`...

Reply:
Yeah, `Ignore` is not quite appropriate. Renamed to `NotYetImplemented`